apis/pipeline/pipeline.yaml

```diff
@@ -4494,6 +4494,46 @@ components:
                     type: integer
                     example: 5
 
+        AWSAuthConfig:
```

Comment:
Further thinking about it: maybe this should be called `EKSAuthConfig` since it's actually EKS specific. WDYT @colin014 @bonifaido @pregnor

Reply:
I agree it is used in AWS/EKS context only and not other AWS contexts such as PKE, so it may slightly improve clarity to call it EKS.